### PR TITLE
Add version and changelog files

### LINKED ISSUE #97: Create the release r1.1 of the RM documentation
**Problem description**
The RM documentation in GitHub should also be under release management. 

**Expected action**

- Create a r1.1 now based on issue #35 with the last larger change of the repository ("Update public release name") which was merged on July 4th (shortly before the release candidates of Commonalities and ICM). That could be candidate for r1.1 (if using a version it would be something as 0.1.0-rc) but as we haven't give the artefacts a version we might not need that).

**Additional context**
All changes in GitHub need to be reflected also in the RM Wiki pages to remain in synch.

### PR DESCRIPTION
**Pull request by tanjadegroot**
#### What type of PR is this?

Add one of the following kinds:
* documentation

#### What this PR does / why we need it:

Adds a VERSION.md file for versioning of the Release Mgmt artifacts.
Adds a CHANGELOG.md file for tracking the changes to the Release Mgmt artifacts per release.

#### Which issue(s) this PR fixes:

It contributes to preparing the r1.1

Fixes #97

### CODE REVIEW

**Comment by hdamker** [comment]
@tanjadegroot I created a [draft release](https://github.com/camaraproject/ReleaseManagement/releases/edit/untagged-9300ddf8bc258c7ce76e) for r1.1. There you can find also the auto-generated list of PRs which went into it.

Steps I used to created it: https://github.com/orgs/community/discussions/23535#discussioncomment-3240768

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+It contains the Release Management approved documents in the **documentation** folder:
+   - **[API_Release_Guidelines.md](https://github.com/camaraproject/ReleaseManagement/blob/r1.1/documentation/API_Release_Guidelines.md)** - An extract of the API Release Management process form the CAMARA wiki.
+   - **[API-Readiness-Checklist.md](https://github.com/camaraproject/ReleaseManagement/blob/r1.1/documentation/API-Readiness-Checklist.md)** - The checklist to ensure release readiness of the API according to its release type (alpha, release-candidate, public).
+
+### Added
+* Add API-Readiness-Checklist.md to RM project by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/29
+* Add API Release Guidelines to GitHub - extract of the API Release Pro… by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/31
+* Update public release name by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/35
+* Update readme.md and further links in rm project material by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/42
```
Comment (hdamker) [inline CHANGELOG.md:25]: Based on the auto-generated changelist from GitHub:

```suggestion
* Add API-Readiness-Checklist.md to RM project by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/29
* Add API Release Guidelines to GitHub - extract of the API Release Pro… by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/31
* Added changelog template and example by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/36 and @hdamker in https://github.com/camaraproject/ReleaseManagement/pull/38
* Update public release name by @tanjadegroot in https://github.com/camaraproject/ReleaseManagement/pull/35
```
#42 was merged end of July.

**Comment by hdamker** [comment]
@tanjadegroot My suggestion would be to close this PR and create a new one with only the content of the proposed CHANGELOG and create the release then on the `temp-r1.1` branch. I can take that if you prefer.

**Comment by tanjadegroot** [comment]
OK to close this PR, as obsolete by now.

**Pull request closed without merging**